Adapt to new signature of executeHttpRequest function

fwilhe opened the pull request:
## What Has Changed?

Updated docs to match [current implementation](https://github.com/SAP/cloud-sdk-js/blob/main/packages/http-client/src/http-client.ts#L351) of the function.

FrankEssenberger reviewed (approved):
LGTM - I reviewed a PR from @deekshas8 where this was also adjusted. Is this already merged or is there a conflict?

fwilhe commented:
> LGTM - I reviewed a PR from @deekshas8 where this was also adjusted. Is this already merged or is there a conflict?

no i was not aware @deekshas8 also changed this
